less/category.less

```diff
@@ -130,13 +130,24 @@
 		color: @gray-light;
 	}
 
-	h2.title .description {
-		font-size: 65%;
+	.description {
+		font-size: 11.7px;
```

Comment:
Instead of a pixel value, can you use `rem`s?

Reply:
Sure